thoth-api/src/contributor/model.rs

```diff
@@ -4,6 +4,19 @@ use uuid::Uuid;
 #[cfg(feature = "backend")]
 use crate::schema::contributor;
 
+#[derive(juniper::GraphQLEnum)]
```

Comment:
This is now done. As discussed, many foreign keys are already exposed in the GraphQL schema (tracked by https://github.com/thoth-pub/thoth/projects/4#card-53206255), so I updated the enums to reflect the current schema, rather than remove all foreign keys entirely.